### PR TITLE
Add empty file cars.txt

### PR DESCRIPTION
**Pull request by MyTwoCount**
Please check this work.
Then you.

### CODE REVIEW

File: cars.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (MyTwoCount) [inline cars.txt:1]: Can I must add some brand car to this file?

Comment (LittlesProgramer) [inline cars.txt:1]: No, it must to be empty file.
Great work.

Comment (MyTwoCount) [reply cars.txt:1]: Can I close this issue ?.

Comment (LittlesProgramer) [reply cars.txt:1]: No, I close it.

**Review by LittlesProgramer — COMMENTED** [review]
close #11

**Review by LittlesProgramer — COMMENTED** [review]
close #11

**Pull request merged**